File: VB/Scaffolding.CustomFilter/Data/Issue.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.ComponentModel.DataAnnotations
Namespace Scaffolding.CustomFilter.Model
    Public Class Issue
        Public Property ID() As Integer
        Public Property IssueType() As IssueType
        Public Property Subject() As String
        Public Property IsUrgent() As Boolean
        Public Property CreatedOn() As Date
    End Class
End Namespace

File: CS/Scaffolding.CustomFilter/Data/Issue.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
namespace Scaffolding.CustomFilter.Model {
    public class Issue {
        public int ID { get; set; }
        public IssueType IssueType { get; set; }
        public string Subject { get; set; }
        public bool IsUrgent { get; set; }
        public DateTime CreatedOn { get; set; }
    }
}